WeHere/Views&ViewModels/Components/Modal/BottomModal/BottomModal.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  BottomModal.swift
+//  WeHere
+//
+//  Created by Chanhee Jeong on 2022/11/21.
+//
+
+import SwiftUI
+
+struct BottomModal: View {
+    @State private var currentIndex = 0
+    let place = Place.mockData[1]
+    
+    var body: some View {
+        TabView(selection: $currentIndex) {
+            ForEach(0..<2) { index in
+                ZStack {
+                    Color.white
+                    switch index {
+                    case 0:
+                        pokeView
+                            .padding(EdgeInsets(top: 28, leading: 28, bottom: 28, trailing: 28))
+                        
+                    case 1:
+                        placeDetailView
+                            .padding(EdgeInsets(top: 28, leading: 28, bottom: 28, trailing: 28))
+                        
+                    default:
+                        EmptyView()
+                    }
+                }
+                .clipShape(RoundedRectangle(cornerRadius: 30.0,
+                                            style: .continuous))
+            }
+            .padding(.all, 10)
+        }
+        .tabViewStyle(PageTabViewStyle(indexDisplayMode: .never))
+        .frame(width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.width*0.88)
+        .overlay(
+            VStack {
+                Spacer()
+                IndexView(currentIndex: currentIndex)
+                    .padding(.bottom, 28)
+            }
+        )
+    }
+}
+
+extension BottomModal {
+    private var pokeView: some View {
+        VStack(alignment: .leading) {
+            HStack {
+                Text(place.name)
+                    .font(Font.theme.title1)
+                Spacer()
+            }
+            HStack {
+                Image(place.category.name)
+                    .resizable()
+                    .frame(width: 20, height: 24)
+                Text(place.address)
+                    .font(Font.theme.body2)
+                    .foregroundColor(Color.gray02)
+                Spacer()
+            }
+            HStack {
+                Spacer()
+                VStack {
+                    Spacer()
+                    Button {
+                        HapticManager.shared.notification(type: .success)
+                    } label: {
+                        MultipleStrokeHeartView(width: 112, height: 105, isPoked: place.isPoked, image: "profile-girl")
+                    }
+                    .padding(.bottom, 25)
+                    Text(place.isPoked
+                         ? "푸린님이 콕 한 장소에요!"
+                         : "폼폼님께 이 장소를 콕 하세요!")
+                        .font(Font.theme.body2)
+                        .foregroundColor(Color.gray02)
+                    Spacer()
+                }
+                Spacer()
+            }
+            Spacer()
+        }
+    }
```

Comment:
```suggestion
private var pokeView: some View {
        VStack {
            HStack {
                Text(place.name)
                    .font(Font.theme.title1)
                Spacer()
            }
            HStack {
                Image(place.category.name)
                    .resizable()
                    .frame(width: 20, height: 24)
                Text(place.address)
                    .font(Font.theme.body2)
                    .foregroundColor(Color.gray02)
                Spacer()
            }
            
            Button {
                HapticManager.shared.notification(type: .success)
            } label: {
                MultipleStrokeHeartView(width: 112, height: 105, isPoked: place.isPoked, image: "profile-girl")
            }
            .padding(.bottom, 25)
            
            Text(place.isPoked
                 ? "푸린님이 콕 한 장소에요!"
                 : "폼폼님께 이 장소를 콕 하세요!")
            .font(Font.theme.body2)
            .foregroundColor(Color.gray02)
        }
    }
```
[P1] 하나의 뷰에 필요 이상의 객체를 너무 많이 들고 있다는 생각입니다. 위와 같이 제안합니다. 
더 줄일수 있다면 Best `placeDetailView`도 역시 필요이상의 너무 많은 객체가 있다는 생각 입니다.

Reply:
오 이게 되네요 좋은 지적 감사합니다..!
스유 뷰 그릴때 한번 더 고려해보겠습니다
e143cbfdf17d743f90729a2aa1076da82308f612